products/docker-engine.md

```diff
@@ -17,7 +17,7 @@ auto:
   -   git: https://github.com/moby/moby.git
       regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-ce)?$
 
-# For EOL, see open milestones on https://github.com/moby/moby/milestones.
+# For EOL, see https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md
```

Comment:
The `releasePolicyLink`. must be updated with https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md too.